src/sql/parts/dashboard/pages/serverDashboardPage.contribution.ts

```diff
@@ -77,7 +77,7 @@ let defaultVal = [
 	{
 		name: 'Tasks',
 		widget: {
-			'tasks-widget': {}
+			'tasks-widget': [{ name: 'backup', when: '!mssql:iscloud' }, 'configureDashboard', 'newQuery']
```

Comment:
>backup [](start = 29, length = 6)

isn't this suppose to be restore? I am not sure. Please verify with Karl or Eric

Reply:
SSMS shows restore at the server level

---
In reply to: [172697411](https://github.com/Microsoft/sqlopsstudio/pull/849#discussion_r172697411) [](ancestors = 172697411)

Reply:
@erickangMSFT can you confirm? what pages are restore and backup supposed to appear on.

Reply:
on-prem:
- Server Dashboard: Restore and Backup
- Database Dashboard: Restore and Backup

Azure
- Server Dashboard: none
- Database Dashboard: Restore in the future with Restore dialog support for Azure. Until then none.